src/common/utils.ts

```diff
@@ -73,3 +73,21 @@ export function findAccountByAddressOrName (accounts: EntropyAccountConfig[], al
     accounts.find(account => account.name === aliasOrAddress)
   )
 }
+
+export function formatDispatchError (dispatchError) {
+  let msg: string
+  if (dispatchError.isModule) {
+    // for module errors, we have the section indexed, lookup
+    const decoded = this.entropy.substrate.registry.findMetaError(
+      dispatchError.asModule
+    )
+    const { docs, name, section } = decoded
+
+    msg = `${section}.${name}: ${docs.join(' ')}`
+  } else {
+    // Other, CannotLookup, BadOrigin, no extra info
+    msg = dispatchError.toString()
+  }
+
+  return Error(msg)
+}
```

Comment:
I saw this code replicated across code base. 

TODO: rats I just noticed it uses "this" need to pass in `entropy`
```suggestion
export function formatDispatchError (entropy, dispatchError) {
  let msg: string
  if (dispatchError.isModule) {
    // for module errors, we have the section indexed, lookup
    const decoded = entropy.substrate.registry.findMetaError(
      dispatchError.asModule
    )
    const { docs, name, section } = decoded

    msg = `${section}.${name}: ${docs.join(' ')}`
  } else {
    // Other, CannotLookup, BadOrigin, no extra info
    msg = dispatchError.toString()
  }

  return Error(msg)
}
```